client/app/components/Modal/__tests__/Modal.spec.jsx

```diff
@@ -39,19 +42,24 @@ describe('Modal', () => {
     jest.spyOn(window, 'alert');
   });
 
+  afterEach(() => {
+    jest.restoreAllMocks();
+  });
+
```

Comment:
Seems like before we don't restore the mocks so `toHaveBeenCalled` was not actually a useful assertion.